### PR TITLE
Made plugin get head textures async

### PR DESCRIPTION
**Pull request by PeachesMLG**
As an alternative to https://github.com/Iridium-Development/IridiumSkyblock/pull/191

### CODE REVIEW

**Comment by PeachesMLG** [comment]
This commit is just a few reformattings and calling addContent async, you can see most changes here: https://github.com/Iridium-Development/IridiumCore/commit/53b77cc1b0fe7fbda0b184cc875fbbf38ac40305

File: src/main/java/com/iridium/iridiumskyblock/configs/inventories/IslandRanksInventoryConfig.java [hunk 1]
```diff
@@ -2,25 +2,8 @@
...
-    public int size;
-    /**
-     * The title of the GUI
-     */
-    public String title;
-    /**
-     * The background of the GUI
-     */
-    public Background background;
+public class IslandRanksInventoryConfig extends NoItemGUI{
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/configs/inventories/IslandRanksInventoryConfig.java:6]: There is no space between NoItemGUI and the paranthese in all of these classes

File: src/main/java/com/iridium/iridiumskyblock/configs/inventories/IslandRanksInventoryConfig.java [hunk 2]
```diff
@@ -42,4 +25,14 @@
...
+        this.size = size;
+        this.title = title;
+        this.background = background;
+        this.owner = owner;
+        this.coOwner = coOwner;
+        this.moderator = moderator;
+        this.member = member;
+        this.visitor = visitor;
+    }
 }
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/configs/inventories/IslandRanksInventoryConfig.java:38]: Missing newline

**Review by dlsf — CHANGES_REQUESTED** [review]

**Comment by dlsf** [comment]
Codacy also has an issue

**Pull request merged**